app/js/views/app_view.js

```diff
@@ -1,20 +1,22 @@
 define([
   'jquery',
   'backbone',
-  'text!templates/app_view.ejs'
-], function ($, Backbone, indexTemplate) {
+  'views/new_todo_view'
+
+], function ($, Backbone, ToDoView) {
   return Backbone.View.extend({
     el: $('#todo-app'),
+    events: {
+      'click #newToDo': 'onNewTodoClicked'
+    },
 
-    currentDate: function () {
-      return new Date();
+    initialize: function () {
+      this.todoView = new ToDoView();
     },
 
     render: function () {
-      var compiledTemplate = ejs.render(indexTemplate, {view: this, model: this.model}, {});
-      this.$el.empty();
-      this.$el.append(compiledTemplate);
+      this.$el.append(this.todoView.render().$el);
```

Comment:
I can update this line like this `this.todoView.render().$el` and all the tests are still passing.